### PR TITLE
flex-col

### PR DESCRIPTION
**Pull request by billythedummy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3552,17 +3553,14 @@ <h1 id="title" class="text-xl font-bold mx-auto truncate w-full">
...
           id="view"
         ></div>
         <!-- End chat/rooms list -->
 
         <!-- Start message input row -->
-        <footer
-          id="message_form"
-          class="items-end gap-x-2 hidden w-full z-10 bg-white"
-        >
+        <footer id="message_form" class="gap-x-2 hidden w-full z-10 bg-white">
```
Comment (billythedummy) [inline index.html:3563]: removing `items-end` resolves https://github.com/igneous-labs/omnispace/pull/67#discussion_r1121135894

File: viewportHax.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
-function setViewHeight() {
-  console.log("called");
+function setMainHeight() {
+  // console.log("called");
```
Comment (billythedummy) [inline viewportHax.js:2]: just some refactoring here and commenting out console.log

File: index.html [hunk 3]
```diff
@@ -46,15 +46,16 @@
     <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
     <link href="/dist/output.css" rel="stylesheet" />
   </head>
-  <!-- overflow-y-hidden doesnt work with safari -->
+  <!-- overflow-y-hidden doesnt work with safari, only overflow-hidden -->
   <body
     x-data="{
       characterMenuOpen: false,
     }"
+    class="overflow-hidden flex flex-col"
```
Comment (billythedummy) [inline index.html:54]: re-added overflow-hidden here, this was supposed to disable scroll on ios as well, but not wiggle

File: index.html [hunk 4]
```diff
@@ -46,15 +46,16 @@
...
   >
     <nav
       id="nav"
-      class="flex items-center px-4 py-2 gap-2 sticky top-0 bg-white z-10"
+      class="flex items-center px-4 py-2 gap-2 sticky bg-white z-10"
```
Comment (billythedummy) [inline index.html:58]: removed `top-0` here since this elem wasnt positioned so it shouldnt do anything?

**Review by Johnnycus — APPROVED** [review]

**Pull request merged**